<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QMainWindow" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenPrincipale</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widgetCentral">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0" columnstretch="1,0,0">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>142</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 232, 232);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLCDNumber" name="lcd_hour">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>42</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_min">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="intValue" stdset="0">
            <number>42</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_sec">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="intValue" stdset="0">
            <number>42</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,1,1,0,15">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="b_tb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>65</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Tableau de bord</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_graph">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Graphiques</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_carte">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Carte</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_console">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Console</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Historique</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_param">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Param√®tres</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QVBoxLayout" name="chronolayout"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QStackedWidget" name="stack">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="p_tabl_bord">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="container"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_debug_console">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPlainTextEdit" name="console">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_historique">
         <layout class="QGridLayout" name="verticalLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QTabWidget" name="tab_historique"/>
            </item>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="actualizeTableButton">
                <property name="text">
                 <string>Actualiser</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="time_end">
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>10</minute>
                  <second>0</second>
                  <year>2000</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::HourSection</enum>
                </property>
                <property name="time">
                 <time>
                  <hour>0</hour>
                  <minute>10</minute>
                  <second>0</second>
                 </time>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="time_start"/>
              </item>
              <item>
               <widget class="QCheckBox" name="check_all_values">
                <property name="text">
                 <string>(ne marche pas)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_params">
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0">
          <item row="3" column="2">
           <widget class="QLineEdit" name="dataServerLineEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>URL du serveur de r√©ception</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QCheckBox" name="interpol_curve">
            <property name="text">
             <string>Interpoler les valeurs de courbe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QCheckBox" name="get_infos">
            <property name="text">
             <string>Activer mise √† jour des infos</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QTimeEdit" name="heureLancement">
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Heure de lancement</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_graphics">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="7,0,0,0">
          <item row="2" column="0">
           <widget class="QMdiArea" name="zone_graph"/>
          </item>
          <item row="3" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="sel_capteur"/>
             </item>
             <item>
              <widget class="QComboBox" name="sel_valeur"/>
             </item>
             <item>
              <widget class="QPushButton" name="add_graph">
               <property name="text">
                <string>Ajouter le graphique</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_optimiser">
               <property name="text">
                <string>OPTIMISATION</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="bowltext">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>1 minute</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_maps">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="c_maps"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="barreStatus"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
